plugins/CoreHome/vue/src/ReportingMenu/ReportingMenu.vue

```diff
@@ -286,7 +286,9 @@ export default defineComponent({
         }
       });
 
-      WidgetsStoreInstance.reloadAvailableWidgets();
+      WidgetsStoreInstance.reloadAvailableWidgets().then(() => {
+        Matomo.postEvent('ReportingMenu.availableWidgetsReloaded');
```

Comment:
Should we dispatch this event as `WidgetsStore.reloaded` inside  `WidgetsStore.reloadAvailableWidgets()`?

Then we would have it a bit more decoupled from the `ReportingMenu`, and it would be less confusing where the data in the dashboard is coming from :thinking:

Reply:
@mneudert I had considered something along those lines. I made the change.